incubator/java-openliberty/templates/default/src/test/java/it/dev/appsody/starter/HealthEndpointTest.java

```diff
@@ -92,7 +92,7 @@ private Response getResponse(String url) {
      *          - response received from the target URL.
      */
     private void assertResponse(String url, Response response) {
-        assertEquals("Incorrect response code from " + url, 200, response.getStatus());
+        assertEquals(200, response.getStatus(), "Incorrect response code from " + url);
     }
 
 }
```

Comment:
Should we add, in the README, a quick general note.. "hey, you might be more used to JUnit 4 but Junit 5 is cool" and link to some resources like:  https://developer.ibm.com/dwblog/2017/top-five-reasons-to-use-junit-5-java/   (or Billy Korando's articles)?    Just a sentence.

Reply:
Good idea